tests/test_audit.py

```diff
@@ -19,6 +19,8 @@ def audit_extractor() -> GithubAuditLogExtractor:
         per_page=DEFAULT_PER_PAGE,
         enterprise_name="test-enterprise",
         actions=["protected_branch.create"],
+        actors=["test-include-actor"],
+        exclude_actors=["test-exclude-actor"],
```

Comment:
You should have tests that exercise: 

- different `actions` (at least None, empty, 1, 2)
- different `actors` (None, empty, 1, 2)
- different `exclude_actors` (None, empty, 1, 2)

And also potentially different combinations as well.  You may want to look into parameterized testing for this.